CeeFM/Decideify
# Posts are now rendering on Social.js, but no comments or suggestion tag functionality just yet

**Description**
You can now add posts to the database, and then also attach a suggestion to that post! The Post will render on the social.js page after reloading, but the suggestion and comments are not there just yet. Soon though!

**Type of Change**


**Testing Instructions**

- Clone this project’s repository to your machine
  -If you’ve already done this and are currently working on a different branch of the project, be sure to `git add` and `git commit` whatever you’re working on on your branch, before you attempt to switch branches
  -Then `git checkout main` to switch to the project’s main branch)
- Navigate to the root directory for this repo (yourParentFolderName/Decideify)
- `git fetch -a` to retrieve the current remote-tracked branches for the project, you should see a branch named `cm-suggestions` as one of the available branches
- `git checkout cm-suggestions` to switch to my branch
- If you don't see the most recent version of my files, `git pull origin cm-suggestions`
- `Start Decideify.sln` to open the project in Visual Studio
- If you haven’t created the Decideify database, please do so in Visual Studio and use the following SQL scripts to populate it accordingly:
  - Start a new query in your database and copy in this SQL script before running to fill out tables and columns: [First SQL Script for Creating DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Create_DB.sql)
  - ctrl + a and delete what is in that query and replace it with this SQL script before running to seed the database: [Second SQL Script for Seeding DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Seed_Users_DB.sql)

- Click the execute with debugger solid green button to launch the app
- On social.js you can:
  - Fill out the post form, submit it, and it adds to the database
  - You can pick a suggestion to add to that post from a modal that pops up thereafter, or you can continue. Either way the page will reload.
  - Your post will render as the top post on that page after the reload. The suggestions and any related comments are not rendering yet, even though the suggestions are being added if you tag one during post creation. Comments haven't even been created or added to database yet though.

Checklist
